tap_github/schemas/project_cards.json

```diff
@@ -1,108 +1,126 @@
 {
-    "type": [
-      "null",
-      "object"
-    ],
-    "properties": {
-      "_sdc_repository": {
-        "type": [
-          "null",
-          "string"
-        ]
-      },
-      "cards_url": {
-        "type": [
-          "null",
-          "string"
-        ]
-      },
-      "name": {
-        "type": [
-          "null",
-          "string"
-        ]
-      },
-      "url": {
-        "type": [
-          "null",
-          "string"
-        ]
-      },
-      "id": {
-        "type": [
-          "null",
-          "number"
-        ]
-      },
-      "node_id": {
-        "type": [
-          "null",
-          "string"
-        ]
-      },
-      "note": {
-        "type": [
-          "null",
-          "string"
-        ]
-      },
-      "creator": {
-        "type": [
-          "null",
-          "object"
-        ],
-        "properties": {
-          "login": {
-            "type": [
-              "null",
-              "string"
-            ]
-          },
-          "id": {
-            "type": [
-              "null",
-              "number"
-            ]
-          }
+  "type": [
+    "null",
+    "object"
+  ],
+  "properties": {
+    "_sdc_repository": {
+      "type": ["null", "string"]
+    },
+    "project_id": {
+      "type": ["null", "string"]
+    },
+    "column_name": {
+      "type": ["null", "string"]
+    },
+    "cards_url": {
+      "type": ["null", "string"]
+    },
+    "name": {
+      "type": ["null", "string"]
+    },
+    "url": {
+      "type": ["null", "string"]
+    },
+    "id": {
+      "type": ["null", "number"]
+    },
+    "node_id": {
+      "type": ["null", "string"]
+    },
+    "note": {
+      "type": ["null", "string"]
+    },
+    "creator": {
+      "type": [
```

Comment:
Creator we used at many places, can we move this to common?

Reply:
Creator is same as user, hence used it